package.json

```diff
@@ -60,7 +58,7 @@
     "setupTestFrameworkScriptFile": "<rootDir>/app/javascript/test/setup.js"
   },
   "scripts": {
-    "lint": "standard $(find config webpack -name *.js)",
+    "lint": "eslint $(find app/javascript -name *.js -or -name *.jsx)",
```

Comment:
Maybe `eslint app/javascript --ext js,jsx`?

Reply:
Nice, didn't know about `--ext`. Fixed in f830f3d